docs/reference-sequence-import.md

```diff
@@ -0,0 +1,124 @@
+            <IDENTIFIERS>
+                <PRIMARY_ID>GCA_000001405.2</PRIMARY_ID>
+                <SUBMITTER_ID namespace="Genome Reference Consortium">GRCh37.p1</SUBMITTER_ID>
+            </IDENTIFIERS>
+            <TITLE>Genome Reference Consortium Human Build 37 patch release 1 (GRCh37.p1)</TITLE>
+            <DESCRIPTION>The human reference assembly defines a standard upon which other whole genome studies are based. Providing the best representation of the human genome is essential for facilitating continued progress in understanding and improving human health. &lt;p&gt;The Human Genome Project (HGP) was an international research collaboration coordinated by the U.S. Department of Energy (DOE) and National Institutes of Health (NIH) whose goals were to determine the sequence of the human chromosomes and identify and map all human genes. The project began in 1988 when Congress funded both the NIH and the DOE and was completed in April of 2003. The HGP used a clone-oriented approach to produce map data, clone reagents, and the assembled human genome sequence. The International Human Genome Sequencing Consortium published the first draft of the human genome, with the sequence 90% complete, in the journal Nature in February 2001. The full sequence was completed in April 2003. The HGP resulted in significant technology, tool, and resource development that continues to have a significant impact on medicine and other life sciences. &lt;/p&gt; &lt;p&gt;The Genome Reference Consortium (GRC) was formed in 2008 to maintain the reference assembly for the human genome. The goals of this group are to correct regions that are misrepresented, to close remaining gaps, and to produce alternative assemblies of structurally variant loci. The consortium does experimental work to address gaps or sub-optimal sequence regions and has developed the infrastructure to review and curate assembly joins. The GRC consists of: The Wellcome Trust Sanger Institute, The Genome Center at Washington University, The European Bioinformatics Institute and The National Center for Biotechnology Information. The public can see regions under review and report genome problems at the GRC website, &lt;a href="http://genomereference.org" alt="link to GRC home page"&gt;http://genomereference.org&lt;/a&gt;. &lt;/p&gt; &lt;p&gt;The human genome assembly submitted by the GRC is available in GenBank under accession ranges GL000001-GL000258 (the scaffolds), and CM000663-CM000686 (the chromosomes).&lt;br/&gt;&lt;p&gt;&lt;b&gt;Comment&lt;/b&gt;&lt;br/&gt;The DNA sequence is composed of genomic sequence, primarily finished clones that were sequenced as part of the Human Genome Project. PCR products and WGS shotgun sequence have been added where necessary to fill gaps or correct errors. All such additions are manually curated by GRC staff. For more information see: http://genomereference.org</DESCRIPTION>
```

Comment:
The description takes too much space; I think we should redact it for readability purposes. After all, this is just an example.
```suggestion
            <DESCRIPTION>The human reference assembly defines a standard upon which other whole genome studies are based. Providing the best representation of the human genome is essential for facilitating continued progress in understanding and improving human health. [...]</DESCRIPTION>
```